project-metrics/metrics_service/database/models.py

```diff
@@ -120,11 +120,12 @@ def __repr__(self) -> Text:
                                    self.commit_hash)
 
 
-class PullRequestStatus(enum.Enum):
+class CommitStatus(enum.Enum):
```

Comment:
Why is this renamed?

Reply:
It _was_ renamed because it seemed like commits had the status, not the PR; but as determined offline that's a whole separate data model/API endpoint. I'll revert this.